packages/middleware-signing/src/configurations.ts

```diff
@@ -32,6 +32,29 @@ export interface AwsAuthInputConfig {
    */
   signingRegion?: string;
 }
+
+export interface CustomAwsAuthInputConfig {
```

Comment:
Why is this needed?

Reply:
For non-AWS service, we don't need signingRegion as a config input. The region acts as the signingRegion.